src/pages/form/basic-form/index.tsx

```diff
@@ -0,0 +1,263 @@
+                  style={{ minHeight: 32 }}
+                  placeholder={formatMessage({ id: 'formandbasic-form.goal.placeholder' })}
+                  rows={4}
+                />,
+              )}
+            </FormItem>
+            <FormItem
+              {...formItemLayout}
+              label={<FormattedMessage id="formandbasic-form.standard.label" />}
+            >
+              {getFieldDecorator('standard', {
+                rules: [
+                  {
+                    required: true,
+                    message: formatMessage({ id: 'formandbasic-form.standard.required' }),
+                  },
+                ],
+              })(
+                <TextArea
+                  style={{ minHeight: 32 }}
+                  placeholder={formatMessage({ id: 'formandbasic-form.standard.placeholder' })}
+                  rows={4}
+                />,
+              )}
+            </FormItem>
+            <FormItem
+              {...formItemLayout}
+              label={
+                <span>
+                  <FormattedMessage id="formandbasic-form.client.label" />
+                  <em className={styles.optional}>
+                    <FormattedMessage id="formandbasic-form.form.optional" />
+                    <Tooltip title={<FormattedMessage id="formandbasic-form.label.tooltip" />}>
+                      <LegacyIcon type="info-circle-o" style={{ marginRight: 4 }} />
+                    </Tooltip>
+                  </em>
+                </span>
+              }
+            >
+              {getFieldDecorator('client')(
+                <Input
+                  placeholder={formatMessage({ id: 'formandbasic-form.client.placeholder' })}
+                />,
+              )}
+            </FormItem>
+            <FormItem
+              {...formItemLayout}
+              label={
+                <span>
+                  <FormattedMessage id="formandbasic-form.invites.label" />
+                  <em className={styles.optional}>
+                    <FormattedMessage id="formandbasic-form.form.optional" />
+                  </em>
+                </span>
+              }
+            >
+              {getFieldDecorator('invites')(
+                <Input
+                  placeholder={formatMessage({ id: 'formandbasic-form.invites.placeholder' })}
+                />,
+              )}
+            </FormItem>
+            <FormItem
+              {...formItemLayout}
+              label={
+                <span>
+                  <FormattedMessage id="formandbasic-form.weight.label" />
+                  <em className={styles.optional}>
+                    <FormattedMessage id="formandbasic-form.form.optional" />
+                  </em>
+                </span>
+              }
+            >
+              {getFieldDecorator('weight')(
+                <InputNumber
+                  placeholder={formatMessage({ id: 'formandbasic-form.weight.placeholder' })}
+                  min={0}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 0](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)